### DISCUSSION TITLE
Using multi ranger with cflib to adjust safe distance from wall

### DISCUSSION THREAD

**Discussion by JohnOng123** [bitcraze/discussions#1188] [Q&A]
Hi,
How do I read the multi ranger data and use the data to position itself a certain distance from any obstacle it detects using the cflib python library?
Thanks!

**Comment by knmcguire** [comment]
hi! Have found the multiranger push example already in the cflib? https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/multiranger/multiranger_push.py
